### PR TITLE
Element Properties and Attributes

### PR DESCRIPTION
**Pull request by ruthstocks**
- Edited the content.
- I found the original document quite hard to understand, and hope I haven't misinterpreted anything. 
- I don't think the Using Attributes Vs. Properties section (previously a Warning box) is very helpful without further elaboration. I scanned component classes and component documentation (HTML and Java) and didn't find clear reference or distinction in this regard. Perhaps I am missing something.

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/flow-and-components-documentation/492)

### CODE REVIEW

File: documentation/element-api/tutorial-properties-attributes.asciidoc [hunk 1]
```diff
@@ -115,18 +143,19 @@ element.getStyle().remove("background-color");
...
 ----
 
-[NOTE]
-You cannot set the element's _style_ attribute using the `element.setAttribute` method.
-However you can use `element.getAttribute('style')` to get all the style rules in a single concatenated string.
+Note:
 
-[[textContent]]
-== TextContent
+* You cannot set an element's `style` attribute using the `element.setAttribute` method.
```
Comment (mstahv) [inline documentation/element-api/tutorial-properties-attributes.asciidoc]: Let's remove this bullet, that works just fine, although it is easier to use the getStyle helper.

File: documentation/element-api/tutorial-properties-attributes.asciidoc [hunk 2]
```diff
@@ -115,18 +143,19 @@ element.getStyle().remove("background-color");
...
 
-[NOTE]
-You cannot set the element's _style_ attribute using the `element.setAttribute` method.
-However you can use `element.getAttribute('style')` to get all the style rules in a single concatenated string.
+Note:
 
-[[textContent]]
-== TextContent
+* You cannot set an element's `style` attribute using the `element.setAttribute` method. 
+* You can use the `element.getAttribute('style')` method to get all style rules in a single concatenated string.
```
Comment (mstahv) [inline documentation/element-api/tutorial-properties-attributes.asciidoc]: As we remove the above, this and the whole Note section becomes obsolete. So let's remove this line and the above "Note:" too

File: documentation/element-api/tutorial-properties-attributes.asciidoc (hunk 1 above)
Comment (ruthstocks) [reply documentation/element-api/tutorial-properties-attributes.asciidoc]: removed

File: documentation/element-api/tutorial-properties-attributes.asciidoc (hunk 2 above)
Comment (ruthstocks) [reply documentation/element-api/tutorial-properties-attributes.asciidoc]: removed

**Pull request merged**